javascript/javascript2/week1/spritAnimal.js

```diff
@@ -0,0 +1,45 @@
+
+
+const spiritName = ["The White Butterfly" , "The Funny Dragon" , " The Crazy Worm" , " The Sleepy Snail" , " The Lovely Panda" , "  The Wet Seadog" , " The Huge Tiger" , " The Awesome Chameleon" , " The Kindly Siberian Husky" , " The Beautiful Fresian Horse"]
+
+
+const enterNameInput = document.getElementById('name');
+
+//get random spirit name 
+function getspiritName() {
+
+const random = Math.floor(Math.random() * spiritName.length);
+console.log(`${massage.value} -${spiritName[random]}`)
+
+
+function myFunction() {
```

Comment:
It is a good idea when the user clicks without entering the name, show an alert!

    if (massage.value == "" || massage.value == "null") {
        alert("Enter your name to generate your spirit animal");
   else 
        document.getElementById("getMassage").innerHTML = `${massage.value} -${spiritName[random]}`;